rpmrc.in

```diff
@@ -404,7 +405,7 @@ buildarchtranslate: sh4a: sh4
 
 buildarchtranslate: aarch64: aarch64
 
-buildarchtranslate: riscv: riscv64
+buildarchtranslate: riscv32: riscv64
```

Comment:
This seems wrong. Surely you don't want riscv32 to generate riscv64 packages by default?

Reply:
> This seems wrong. Surely you don't want riscv32 to generate riscv64 packages by default?

yes this should be fixed!